src/main/java/com/ernie/app/ticket/repository/TicketRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.ernie.app.ticket.repository;
+
+import com.ernie.app.ticket.model.Ticket;
+import com.ernie.app.ticket.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface TicketRepository extends JpaRepository<Ticket, UUID> {
+
+    Ticket findByUser(User user);
```

Comment:
this method is not with custom query, add smth like:
` @Query(value = "...")`

Reply:
https://github.com/my20yeezy/Learning-at-Andersen/pull/11/commits/d54d2da88703f123f6b1c858156d373853e2af05